clang/docs/checkedc/3C/INSTALL.md

```diff
@@ -0,0 +1,54 @@
+# Building 3C
+
+This document describes our current setup and build process for 3C.  You may find helpful information about topics not addressed here in the [general Checked C build documentation](../Setup-and-Build.md).  Some items here are potentially applicable to Checked C as a whole, and we hope to eventually harmonize them with the Checked C documentation as appropriate.
+
+## Supported environments
+
+We work primarily on Linux.  Checked C supports Windows, but we do not test 3C regularly on Windows.  Some of us use Mac OS X, which Checked C does not officially support; we have gotten 3C to mostly work with some additional steps (see below), but there are some known problems and [Microsoft is still considering whether to work with us to address them](https://github.com/microsoft/checkedc/issues/424#issuecomment-720648442).
+
+## Basics
+
+**(TODO: Test from start to finish!)**
+
+Assuming you have already cloned the `checkedc-clang` repository, run the following (from the root of the working tree) for a basic build:
+
+```
+# See https://github.com/correctcomputation/checkedc-clang/issues/296 about this.
+(llvm/projects/checkedc-wrapper && git clone https://github.com/microsoft/checkedc)
```

Comment:
Should we change these build instructions to install the checked C headers in `checkedc-wrapper` ? I can't tell if that's what this comment is suggesting. Seems good to me!

Reply:
Ah, it is there. Might be better to remove the issue link entirely and add a quick explanation. It's pretty important to do.

Reply:
Yes, that's what the command does.